### PR TITLE
create example

### PR DESCRIPTION
**Pull request by alexlin519**

### CODE REVIEW

File: src/pycricketpred/modelling.py [hunk 1]
```diff
@@ -188,6 +188,7 @@ def evaluate_model(final_pipe, X_test, y_test, save_image_path):
     conf_mat = metrics.confusion_matrix(y_test, final_pipe.predict(X_test))
     plot_cm = metrics.ConfusionMatrixDisplay(conf_mat)
     plot_cm.plot()
+    ##could make the image name as an input of the function
```
Comment (alexlin519) [inline src/pycricketpred/modelling.py:191]: however, if we change this, we also need to change the tests and our original function in accordingly position.not sure if it is worthy.

File: pyproject.toml [hunk 2]
```diff
@@ -28,6 +28,7 @@ vegafusion-python-embed = "1.6.6"
 myst-nb = {version = "^1.0.0", python = "^3.9"}
 sphinx-autoapi = "^3.0.0"
 sphinx-rtd-theme = "^2.0.0"
+jupyter = "^1.0.0"
```
Comment (alexlin519) [inline pyproject.toml:31]: add jupyter lab packages to poetry

**Review by alexlin519 — COMMENTED** [review]
some comments about the code, not sure if we should updated the code change.

**Comment by Choudoufuhezi** [comment]
looks good, had some merge conflict, but should be good

**Pull request merged**